_posts/2017-12-30-eithersch.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+author: kiralybalint
+date: 2017-12-30 21:01:00 CET
+title: "EitherSCH"
+comment: true
+---
+
+Nincs programod szilveszterre? Írtunk neked egyet..
+
+Természetesen a komoly fejlesztés mellett szükség van néha a kikapcsolódásra, ezért megrendezésre került a Kir-Dev téli tábor. Persze az igazi fejlesztő nem bírja ki fejlesztés nélkül 1 napnál tovább így a második este kissé iluminált állapotban neki áltunk kódolni. A cél az volt, hogy az either.io-hoz hasonló játékot hozzunk létre saját SCH-s kérdésekkel. 
+
+Az eredményt a https://either.sch.bme.hu címen találjátok. Jó szórakozást és boldog új évet kíván a Kir-Dev.
```

Comment:
A linken ssl hiba van. A modern böngészők ezt úgy díjazzák nem engednek be az oldalra es hatalmas betűkkel az arcodba tolják hogy "Insecure connection".

A [letsencrypt](https://letsencrypt.org/) ingyen SSL certet ad és nagyon egyszerű belőni a szerverre. Sőt még automata megújítást is tud.